Gemfile

```diff
@@ -41,6 +41,10 @@ gem 'rake', '10.4.1'
 
 gem 'sass-rails', '4.0.3'
 
+group :production do
```

Comment:
I spoke to @jennyd and she recommended removing the group I added…

Which is the preference?